app/services/books/indexing_strategies/i1/book_docs.rb

```diff
@@ -15,7 +15,7 @@ def docs
           next if page.preface? || page.index?
 
           page.
-            elements(element_classes: DESIRED_ELEMENTS_TO_DOCUMENTS.keys).
+            elements(element_classes: DESIRED_ELEMENTS_TO_DOCUMENTS.keys, omit_by_xpath: "//*[contains(@class, 'os-teacher')]").
```

Comment:
i think it would be more clear and re-usable to have a `remove_elements` that we call before querying out `elements`

Reply:
Should we remove them also by passing classes like for desired elements or is the usage of xpath okay here?
If we want to use classes it would be good to create them in `cnx-ruby`, but then when you want to remove a new element then you have to update 2 repos

Reply:
I've added `remove_elements(xpath: string)`